tests/test_tutorial/test_async_sql_databases/test_tutorial001.py

```diff
@@ -94,38 +210,70 @@
                     "type": {"title": "Error Type", "type": "string"},
                 },
             },
-            "HTTPValidationError": {
-                "title": "HTTPValidationError",
-                "type": "object",
-                "properties": {
-                    "detail": {
-                        "title": "Detail",
-                        "type": "array",
-                        "items": {"$ref": "#/components/schemas/ValidationError"},
-                    }
-                },
-            },
         }
     },
 }
 
 
-def test_openapi_schema():
-    with TestClient(app) as client:
-        response = client.get("/openapi.json")
-        assert response.status_code == 200, response.text
-        assert response.json() == openapi_schema
+note = {"text": "Foo bar", "completed": False}
+completed_note = {"text": "Foo bar", "completed": True}
 
 
-def test_create_read():
+@pytest.fixture(scope="module")
+def client():
     with TestClient(app) as client:
-        note = {"text": "Foo bar", "completed": False}
-        response = client.post("/notes/", json=note)
-        assert response.status_code == 200, response.text
-        data = response.json()
-        assert data["text"] == note["text"]
-        assert data["completed"] == note["completed"]
-        assert "id" in data
-        response = client.get("/notes/")
-        assert response.status_code == 200, response.text
-        assert data in response.json()
+        yield client
+    db_file = Path("./test.db")
+    if db_file.is_file():
+        db_file.unlink()
+
+
+def test_openapi_schema(client):
+    response = client.get("/openapi.json")
+    assert response.status_code == 200, response.text
+    assert response.json() == openapi_schema
+
+
+def test_create(client):
+    response = client.post("/notes/", json=note)
+    assert response.status_code == 200, response.text
+    data = response.json()
+    assert "id" in data
+    assert data["text"] == note["text"]
+    assert data["completed"] == note["completed"]
+
+
+def test_read(client):
+    response = client.get("/notes/1/")
+    assert response.status_code == 200, response.text
+    data = response.json()
+    assert "id" in data
+    assert data["text"] == note["text"]
+    assert data["completed"] == note["completed"]
+    response = client.get("/notes/")
+    assert response.status_code == 200, response.text
+    assert data in response.json()
+
+
+def test_update(client):
+    response = client.put("/notes/1/", json=completed_note)
+    print(
```

Comment:
Remove

Reply:
Yes, it has been removed now, it was originally there for debugging the tests but I forgot to remove it.